equinix/services/fabricv4/api_client.py

```diff
@@ -65,6 +67,7 @@ class ApiClient:
         'bool': bool,
         'date': datetime.date,
         'datetime': datetime.datetime,
+        'decimal': decimal.Decimal,
```

Comment:
Decimal support added in https://github.com/OpenAPITools/openapi-generator/pull/19203